src/sentry/lang/native/symbolicator.py

```diff
@@ -134,6 +135,20 @@ def process_payload(self, stacktraces, modules, signal=None):
         res = self._process("symbolicate_stacktraces", "symbolicate", json=json)
         return process_response(res)
 
+    def process_js(self, stacktraces, modules, dist):
+        source = get_internal_release_file_source(self.project, self.release)
+
+        json = {
+            "source": source,
+            "stacktraces": stacktraces,
+            "modules": modules,
```

Comment:
this also needs the `allow_scraping` setting that we are now able to use

Reply:
Done